### PR TITLE
Add all post-freeze JP content

### LINKED ISSUE #994: [Localization] Add translated content from freeze to the docs site
## Summary 
Once we have the translated content from the freeze, we will need to download it, convert it to mdx and then add it to the docs site. Therefore we should use our migrations scripts again in order to get this content specifically into the docs site. 

## Acceptance Criteria

- [ ] Add all available JP content from the old onelink site that was last updated  prior to our docs freeze to the new docs site
- [ ] All content is added in the correct format (MDX in our i18n folder)
- [ ] report back with any missing JP content relates to #1236

### PR DESCRIPTION
**Pull request by zstix**
## Description
Now that we have translated content form the freeze, we need to convert it to MDX and add it to the repository. This PR includes the new content as well as the script used to turn the unserialized HTML content into the correct MDX format.

## Reviewer Notes
Once the content is correctly formatted and checked in, I'll mark this PR as ready for review. I wanted to put a draft up early in case someone else decided to continue the work.

## Related Issue(s)
* Closes #994

## TODOs
* [x] Turn all the links into local links
* [x] Check that we have all the necessary images
* [x] Keep `#` in urls, if present
* [x] Update all local links to have the new JP basename
* [x] Make sure all image paths are correct (e.g. `imagess`)
* [x] Move the pages into the right directory based on Julien's comment
* [x] MDX pages with imports

### CODE REVIEW

**Comment by nr-kkenney** [comment]
we identified these documents as not having a reference in any of the nav.yaml files:
```json
[
  '/docs/alerts-applied-intelligence/alerts-applied-intelligence.html',
  '/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph.html',
  '/docs/apis/rest-api-v2/api-explorer-v2/introduction-new-relics-rest-api-explorer.html',
  '/docs/apis/rest-api-v2/get-started/introduction-new-relic-rest-api-v2.html',
  '/docs/apis/synthetics-rest-api/monitor-examples/manage-synthetics-monitors-rest-api.html',
  '/docs/browser/browser-monitoring/browser-agent-spa-api.html',
  '/docs/browser/single-page-app-monitoring/get-started/introduction-single-page-app-monitoring.html',
  '/homepage.html'
]
```

**Comment by zstix** [comment]
When we ran the script, we found issues with the following (continuing to look through this):
```
docs-website/src/content/docs/agents/net-agent/net-agent-api.mdx
docs-website/src/content/docs/agents/php-agent/php-agent-api.mdx
docs-website/src/content/docs/agents/python-agent/python-agent-api.mdx
docs-website/src/content/docs/alerts-applied-intelligence/alerts-applied-intelligence.mdx
docs-website/src/content/docs/browser/browser-monitoring/browser-agent-spa-api.mdx
docs-website/src/content/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api.mdx
docs-website/src/content/docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api.mdx
homepage.html
```

**Comment by nr-kkenney** [comment]
@nr-kkenney need to fix links

**Comment by jmiraNR** [comment]
Hello @nr-kkenney & @zstix, my answers:
**Set #1 (the HTML):** here is what I found:
'/docs/alerts-applied-intelligence/alerts-applied-intelligence.html'
=> https://docs.newrelic.com/docs/alerts-applied-intelligence/
'/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph.html'
=> https://docs.newrelic.com/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph/
'/docs/apis/rest-api-v2/api-explorer-v2/introduction-new-relics-rest-api-explorer.html'
=> https://docs.newrelic.com/docs/apis/rest-api-v2/api-explorer-v2/introduction-new-relics-rest-api-explorer/
'/docs/apis/rest-api-v2/get-started/introduction-new-relic-rest-api-v2.html'
=> https://docs.newrelic.com/docs/apis/rest-api-v2/get-started/introduction-new-relic-rest-api-v2/
'/docs/apis/synthetics-rest-api/monitor-examples/manage-synthetics-monitors-rest-api.html'
=> https://docs.newrelic.com/docs/apis/synthetics-rest-api/monitor-examples/manage-synthetics-monitors-rest-api/
'/docs/browser/browser-monitoring/browser-agent-spa-api.html'
=> https://docs.newrelic.com/docs/browser/new-relic-browser/browser-agent-spa-api/
'/docs/browser/single-page-app-monitoring/get-started/introduction-single-page-app-monitoring.html'
=> https://docs.newrelic.com/docs/browser/single-page-app-monitoring/get-started/introduction-single-page-app-monitoring/
'/homepage.html'
=> https://docs.newrelic.com/

**Set #2 (the mdx):** I do see the pages listed. Are you saying we don't have the HTML for these?

**Review by zstix — COMMENTED** [review]
## TODO
* [x] Turn all the links into local links
* [x] Check that we have all the necessary images
* [ ] Keep `#` in urls, if present
* [x] Update all local links to have the new JP basename
* [ ] Make sure all image paths are correct (e.g. `imagess`)
* [ ] Move the pages into the right directory based on Julien's comment

**Comment by jmiraNR** [comment]
@zstix Set #2: pointing out the English pages. Just add /jp/ for Japanese location.
docs-website/src/content/docs/agents/net-agent/net-agent-api.mdx
= jaJP > docs > agents > net-agent > net-agent-api.html
=> https://docs.newrelic.com/docs/agents/net-agent/net-agent-api/

docs-website/src/content/docs/agents/php-agent/php-agent-api.mdx
= jaJP > docs > agents > php-agent > php-agent-api.html
=> https://docs.newrelic.com/docs/agents/php-agent/php-agent-api/

docs-website/src/content/docs/agents/python-agent/python-agent-api.mdx
= jaJP > docs > agents > python-agent > python-agent-api.html
=> https://docs.newrelic.com/docs/agents/python-agent/python-agent-api/

docs-website/src/content/docs/alerts-applied-intelligence/alerts-applied-intelligence.mdx
= jaJP > docs > alerts-applied-intelligence > alerts-applied-intelligence.html
=> https://docs.newrelic.com/docs/alerts-applied-intelligence/

docs-website/src/content/docs/browser/browser-monitoring/browser-agent-spa-api.mdx
= jaJP > docs > browser > browser-monitoring > browser-agent-spa-api.html
=> https://docs.newrelic.com/docs/browser/new-relic-browser/browser-agent-spa-api/

docs-website/src/content/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api.mdx
= jaJP > docs > mobile-monitoring > new-relic-mobile-android > android-sdk-api.html
=> https://docs.newrelic.com/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api/

docs-website/src/content/docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api.mdx
= jaJP > docs > mobile-monitoring > new-relic-mobile-ios > ios-sdk-api.html
=> https://docs.newrelic.com/docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api/

**Comment by nr-kkenney** [comment]
I moved the TODOs into the main PR description, just so i could add comments / refer to it easier.
as an end of day update:

Fixed / more fixes for:
* image urls. there were a couple other problems including url encoding (%20), regex not matching as we wanted for urls containing `image` twice.
* fixed page urls. hashes, query parameters, and the replacing we were doing for the old url vs new one.
* troubleshooting type frontmatter. this is no longer being removed by our script 🙈 

Still remaining that I can clearly point out (which i will add to the todo above):
* mdx pages with imports.

**Comment by nr-kkenney** [comment]
just another EOD update:
* moved over / adjusted most of the files we listed above. the ones still remaining are the API pages. i had assumed we didn't need them / they were auto generated, but that was not correct.
* didn't realize the site was failing to build with the changes we committed 🙈  will need to fix that as well.

**Comment by nr-kkenney** [comment]
alright, this PR is now ready for review (at least from my eyes).

there is one callout which is that the following files were added to this pr, will need some separate (non-trivial) migration work in order to get them correctly into the repo:
```
alerts-applied-intelligence.mdx
android-sdk-api.mdx
browser-agent-spa-api.mdx
ios-sdk-api.mdx
net-agent-api.mdx
php-agent-api.mdx
python-agent-api.mdx
```

this is because while the script did run on the html & produce mdx output, these particular page types have a special structure within the repository, that the script is not able to handle currently.

**Review by roadlittledawn — COMMENTED** [review]
Looks perdy good to me. One question re: the remaining rogue English files in `src/i18n`.

@jmiraNR . is it safe to assume that if there are still English `.mdx` files in the `src/i18n/content/jp/` directories we can remove them? looks like this PR updated some of those files, but not all. 

Here's that [original list of English files](https://gist.github.com/roadlittledawn/c604ea323077953a7e88c6e1b47d0634) I found in `src/i18n`

File: src/i18n/content/jp/docs/agents/net-agent/troubleshooting/no-data-appears-net.mdx [hunk 1]
```diff
@@ -10,6 +10,8 @@ metaDescription: Troubleshooting steps if data does not appear in your app after
 
 [コンテンツ](/docs) / [APMエージェント](/docs/agents) / [.NETエージェント](/docs/agents/net-agent) / [トラブルシューティング](/docs/agents/net-agent/troubleshooting)
 
+# データが表示されない（.NET） [#page-title]
```
Comment (roadlittledawn) [inline src/i18n/content/jp/docs/agents/net-agent/troubleshooting/no-data-appears-net.mdx:13]: i saw a couple of these. don't think this heading is supposed to be there. if you look at the English file, it doesn't have this. and troubleshooting docs really should only have Problem, Solution, Cause as possible headings.

**Review by roadlittledawn — APPROVED** [review]
:shipit:

**Pull request merged**